From 5abceffe7a99fcef7a5c9ff49960e1397170b6d3 Mon Sep 17 00:00:00 2001
From: Andy Nichols <andy.nichols@digia.com>
Date: Mon, 27 May 2013 20:10:40 +0200
Subject: [PATCH] Disable qmlmin

---
 tools/tools.pro | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/tools.pro b/tools/tools.pro
index eda9e85..896334b 100644
--- a/tools/tools.pro
+++ b/tools/tools.pro
@@ -2,7 +2,6 @@ TEMPLATE = subdirs
 qtHaveModule(quick): SUBDIRS += qmlscene qmlplugindump
 qtHaveModule(qmltest): SUBDIRS += qmltestrunner
 SUBDIRS += \
-    qmlmin \
     qmlprofiler \
     qmlbundle
 qtHaveModule(quick):qtHaveModule(widgets): SUBDIRS += qmleasing
-- 
1.8.1.2

